api-reference/listing/sync.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: "Synchronise Listings"
+api: "POST https://api.travtus.com/listing/sync/"
+description: "This endpoint synchronises the listings in the given portfolio with the provided listings."
+---
+
+This endpoint allows you to update, create and delete listings for a portfolio.
+
+The endpoint will attempt to reconcile every listing in the request body with an existing one for the given group and update it with the provided details.
+
+If a listing is not found, a new one will be created.
+
+Listings for the provided portfolio that are not included in the request body will be deleted.
+
+If there are no listings passed in the request body, all listings for the provided porfolio will be deleted.
+
+### Header
+
+<ParamField header="Authorization" type="string" default="none" required>
+  The authentication token for your request
+</ParamField>
+
+### Body
+
+<ParamField body="group_id" type="string" default="none" required>
```

Comment:
does the 'required' line at the end signify that the group_id is required?

Reply:
Yup. It gets marked in the docs as required and it implies that we'll reject requests without a group ID in the Adam API. I don't want someone to accidentally delete all their vacancies because they've called this endpoint with no body, so I'm making the group_id required.

Reply:
ah i see, so in that case if the parameter is required, should there be a default value?

Reply:
Do you think it's confusing that it's currently set to none? What it modifies is the placeholder value in the form field, so at the moment, with it set to none, this is what the users see
<img width="566" alt="image" src="https://user-images.githubusercontent.com/119344265/223788532-0544465a-3d23-4169-9e83-1b65fafac1e9.png">